back/spec/acceptance/initiatives_spec.rb

```diff
@@ -679,7 +679,12 @@
         let(:cosponsor) { create(:user) }
         let(:cosponsor_ids) { [cosponsor.id] }
 
-        example_request 'Update the cosponsors of an initiative' do
+        example 'Update the cosponsors of an initiative' do
+          expect { do_request }
+            .to have_enqueued_job(LogActivityJob)
+            .with(instance_of(CosponsorsInitiative), 'created', @user, instance_of(Integer))
+            .exactly(1).times
+
```

Comment:
New regression test to help ensure we actually generate exactly one specific `LogActivityJob` when a new `cosponsors_initiative` is created via an `initiative` PATCH.